mmdet3d/core/bbox/structures/cam_box3d.py

```diff
@@ -220,11 +225,14 @@ def flip(self, bev_direction='horizontal', points=None):
                 self.tensor[:, 6] = -self.tensor[:, 6]
 
         if points is not None:
-            assert isinstance(points, (torch.Tensor, np.ndarray))
-            if bev_direction == 'horizontal':
-                points[:, 0] = -points[:, 0]
-            elif bev_direction == 'vertical':
-                points[:, 2] = -points[:, 2]
+            assert isinstance(points, (torch.Tensor, np.ndarray, BasePoints))
+            if isinstance(points, (torch.Tensor, np.ndarray)):
+                if bev_direction == 'horizontal':
+                    points[:, 0] = -points[:, 0]
+                elif bev_direction == 'vertical':
+                    points[:, 2] = -points[:, 2]
+            elif isinstance(points, BasePoints):
+                points.flip(bev_direction)
```

Comment:
Is there any case that points are still tensors or arrays?

Reply:
> Is there any case that points are still tensors or arrays?

For back-compatibility, we should keep them for a while.